kpops/cli/main.py

```diff
@@ -131,36 +145,39 @@ def get_step_names(steps_to_apply: list[PipelineComponent]) -> list[str]:
 
 
 def filter_steps_to_apply(
-    pipeline: Pipeline, steps: set[str]
+    pipeline: Pipeline, steps: set[str], filter_type: FilterType
 ) -> list[PipelineComponent]:
-    skipped_steps: list[str] = []
+    log.info("KPOPS_PIPELINE_STEPS is defined.")
```

Comment:
still necessary? I'd say we can remove it or at least change it to debug level

Reply:
I changed it to debug and made it verbose